src/main/java/net/mcreator/ui/modgui/BiomeGUI.java

```diff
@@ -436,7 +436,7 @@ public BiomeGUI(MCreator mcreator, ModElement modElement, boolean editingMode) {
 
 		sbbp5.setBorder(BorderFactory.createTitledBorder(
 				BorderFactory.createLineBorder((Color) UIManager.get("MCreatorLAF.BRIGHT_COLOR"), 2),
-				"Custom tree properties (if selected)", 0, 0, getFont().deriveFont(12.0f),
+				(L10N.t("elementgui.biome.custom_tree_properties")), 0, 0, getFont().deriveFont(12.0f),

```

Comment:
Please fix this redundancy:
(L10N.t("elementgui.biome.custom_tree_properties")) <- extra/redundant parantheses